tests/cli/main.cpp

```diff
@@ -196,253 +196,188 @@ class client_connection
-
-      // verify that the upgrade was successful
-      BOOST_CHECK_PREDICATE( std::not_equal_to<uint32_t>(), (nathan_acct_before_upgrade.membership_expiration_date.sec_since_epoch())(nathan_acct_after_upgrade.membership_expiration_date.sec_since_epoch()) );
-      BOOST_CHECK(nathan_acct_after_upgrade.is_lifetime_member());
-
-      // create a new account
-      graphene::wallet::brain_key_info bki = con.wallet_api_ptr->suggest_brain_key();
-      BOOST_CHECK(!bki.brain_priv_key.empty());
-      signed_transaction create_acct_tx = con.wallet_api_ptr->create_account_with_brain_key(bki.brain_priv_key, "jmjatlanta", "nathan", "nathan", true);
-      // save the private key for this new account in the wallet file
-      BOOST_CHECK(con.wallet_api_ptr->import_key("jmjatlanta", bki.wif_priv_key));
-      con.wallet_api_ptr->save_wallet_file(con.wallet_filename);
-
-      // attempt to give jmjatlanta some bitsahres
-      BOOST_TEST_MESSAGE("Transferring bitshares from Nathan to jmjatlanta");
-      signed_transaction transfer_tx = con.wallet_api_ptr->transfer("nathan", "jmjatlanta", "10000", "1.3.0", "Here are some CORE token for your new account", true);
-
-      // get the details for init1
-      witness_object init1_obj = con.wallet_api_ptr->get_witness("init1");
-      int init1_start_votes = init1_obj.total_votes;
-      // Vote for a witness
-      signed_transaction vote_witness1_tx = con.wallet_api_ptr->vote_for_witness("jmjatlanta", "init1", true, true);
-
-      // generate a block to get things started
-      BOOST_CHECK(generate_block(app1));
-      // wait for a maintenance interval
-      BOOST_CHECK(generate_maintenance_block(app1));
-
-      // Verify that the vote is there
-      init1_obj = con.wallet_api_ptr->get_witness("init1");
-      witness_object init2_obj = con.wallet_api_ptr->get_witness("init2");
-      int init1_middle_votes = init1_obj.total_votes;
-      BOOST_CHECK(init1_middle_votes > init1_start_votes);
-
-      // Vote for a 2nd witness
-      int init2_start_votes = init2_obj.total_votes;
-      signed_transaction vote_witness2_tx = con.wallet_api_ptr->vote_for_witness("jmjatlanta", "init2", true, true);
-
-      // send another block to trigger maintenance interval
-      BOOST_CHECK(generate_maintenance_block(app1));
-
-      // Verify that both the first vote and the 2nd are there
-      init2_obj = con.wallet_api_ptr->get_witness("init2");
-      init1_obj = con.wallet_api_ptr->get_witness("init1");
-
-      int init2_middle_votes = init2_obj.total_votes;
-      BOOST_CHECK(init2_middle_votes > init2_start_votes);
-      int init1_last_votes = init1_obj.total_votes;
-      BOOST_CHECK(init1_last_votes > init1_start_votes);
-
-      // wait for everything to finish up
-      fc::usleep(fc::seconds(1));
-   } catch( fc::exception& e ) {
-      edump((e.to_detail_string()));
-      throw;
-   }
-   app1->shutdown();
+    try
+    {
```

Comment:
Why changed the indentation to 4 white spaces (both above and below)?

Reply:
where can I see BitShares Core coding convention ? I prefer 4 white spaces, what do you prefer ?

Reply:
3 spaces. sorry @abitmore i didnt saw this one.

Reply:
indentation everywhere in BitShares Core code base is prefer to 3 spaces ?

Reply:
yes. 3 spaces everywhere(except p2p code that uses 2 but we don't want to change that now)

Reply:
ok sure. Thank you very much ! Does BitShares Core have code convention ?

Reply:
Can you please change the indentation back to 3 white spaces? Thanks.

We don't have code conventions so far.